openedx/core/djangoapps/profile_images/tests/test_images.py

```diff
@@ -146,24 +147,54 @@ def test_generation(self, image_type, dimensions):
             100: "hundred.jpg",
             1000: "thousand.jpg",
         }
-        mock_storage = mock.Mock()
         with make_uploaded_file(dimensions=dimensions, extension=extension, content_type=content_type) as uploaded_file:
-            with mock.patch(
-                "openedx.core.djangoapps.profile_images.images.get_profile_image_storage",
-                return_value=mock_storage,
-            ):
-                create_profile_images(uploaded_file, requested_sizes)
-                names_and_files = [v[0] for v in mock_storage.save.call_args_list]
-                actual_sizes = {}
-                for name, file_ in names_and_files:
-                    # get the size of the image file and ensure it's square jpeg
-                    with closing(Image.open(file_)) as image_obj:
-                        width, height = image_obj.size
-                        self.assertEqual(width, height)
-                        self.assertEqual(image_obj.format, 'JPEG')
-                        actual_sizes[width] = name
-                self.assertEqual(requested_sizes, actual_sizes)
-                mock_storage.save.reset_mock()
```

Comment:
mock_storage never got used after the call to reset_mock, so it wasn't actually serving a purpose.  I assume it was there from an earlier iteration where multiple calls to the method reused the same mock_storage.